Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -969,7 +969,7 @@ function vtkOpenGLTexture(publicAPI, model) {
         if (j <= model.maxLevel) {
           tempData = invertedData[6 * j + i];
         }
-        if (model._openGLRenderWindow.getWebgl2() && !model.resizable) {
+        if (useTexStorage(dataType)) {
```

Comment:
I don't get that but glad you fixed it...

Reply:
that particular line is just factorization to be consistent with the rest of the file.

The other change is more important.